receiver/flinkmetricsreceiver/metadata.yaml

```diff
@@ -0,0 +1,522 @@
+name: flinkmetricsreceiver
+
+resource_attributes:
+  host:
+    description: The host, used to distinguish different jobmanager metrics.
+    type: string
+  taskmanager_id:
+    description: The taskmanager ID, used to distinguish taskmanager metrics.
+    type: string
+  job_name:
+    description: The job name, used to distinguish job metrics.
+    type: string
+  task_name:
+    description: The task name, used to distinguish tasks metrics.
+    type: string
+  subtask_index:
```

Comment:
Since these appear to be very specific to flink, they'll need to be namespaced accordingly `flink.foo.bar`. Also should use dot delimiter.